presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -136,7 +138,12 @@ public WriteMapping toWriteMapping(Type type)
 
     private static ColumnMapping jsonColumnMapping()
     {
-        return ColumnMapping.sliceMapping(JSON, (resultSet, columnIndex) -> jsonParse(utf8Slice(resultSet.getString(columnIndex))), jsonWriteFunction(), domain -> Domain.all(domain.getType()));
+        return ColumnMapping.sliceMapping(JSON, (resultSet, columnIndex) -> jsonParse(utf8Slice(resultSet.getString(columnIndex))), jsonWriteFunction(), jsonPushdown());
```

Comment:
nit: format with each arg on separate line, for readability
```
ColumnMapping.sliceMapping(
    JSON,
     (resultSet, columnIndex) -> jsonParse(utf8Slice(resultSet.getString(columnIndex))), 
     jsonWriteFunction(),
    jsonPushdown());
```

Reply:
done